widget/ui/src/components/Wallet/Wallet.types.ts

```diff
@@ -14,6 +15,7 @@ export type WalletInfo = {
   image: string;
   type: string;
   showOnMobile?: boolean;
+  blockchainTypes: TransactionType[];
```

Comment:
I think maybe it's better to eliminate the paradigm shift between blockchainType and TransactionType and instead name this variable transactionTypes

Reply:
I disagree because it doesn't give the meaning of transaction type here